docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -68,6 +71,15 @@ to prevent accidental escalation. You can make roles specific to your use case i
 
 Consult the [authorization reference section](/docs/admin/authorization/) for more information.
 
+## Controlling access to the Kubelet
+
+Kubelets expose HTTPS endpoints which give access to data of varying sensitivity, and allow performing operations with varying levels of power on the node and within containers.
```

Comment:
seems a little vague and I don't think it's obvious whether this control is particularly important. We should probably make a stronger case and make it sound more serious. Maybe:

```
Kubelets expose HTTPS endpoints which grant powerful control over the node and containers. By default Kubelets allow unauthenticated access to this API.

Production clusters should enable Kubelet authentication and authorization.
```